Documentation/compatibility/deflatestream-uses-native-apis-for-decompression.md

```diff
@@ -0,0 +1,48 @@
+## DeflateStream uses native APIs for decompression
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+
+Starting with apps that target the .NET Framework 4.7.1, the implementation of decompression in the `T:System.IO.Compression.DeflateStream` class has changed to use native Windows APIs by default. This results in an average performance improvement of approximately 300%. 
```

Comment:
Also improvement is only for decompression (DeflateStream despite confusing name of this class)